schema/swagger.yml

```diff
@@ -804,28 +804,26 @@ paths:
     post:
       tags:
         - Signatures
```

Comment:
Why does the endpoint remain in plural if we now only accept a single signature/transaction?

Reply:
@diego-G I had it singular originally but after a discussion with @nazarhussain it was decided to leave it plural for consistency because we use the plural form everywhere and we wanted all of them (plural and singular) under the same tag. What do you think?

Reply:
Yep, keep `transactions`/`signatures` for consistency.